packages/core/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@api/core",
+  "version": "1.0.0",
+  "description": "The magic behind `api` 🧙",
```

Comment:
Could stand to give this a better description.

Reply:
Honestly I dig it!